models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1797,7 +1797,10 @@ FROM
     ('zro-layerzero', 'ethereum', 'ZRO', 0x6985884c4392d348587b19cb9eaaf157f13271cd, 18),
     ('quartz-sandclock', 'ethereum', 'QUARTZ', 0xba8a621b4a54e61c442f5ec623687e2a942225ef, 18),
     ('wcomai-wrapped-commune-ai', 'ethereum', 'WCOMAI', 0xc78b628b060258300218740b1a7a5b3c82b3bd9f, 18),
-    ('wbeth-wrapped-beacon-eth', 'ethereum', 'WBETH', 0xa2E3356610840701BDf5611a53974510Ae27E2e1, 18)
+    ('wbeth-wrapped-beacon-eth', 'ethereum', 'WBETH', 0xa2E3356610840701BDf5611a53974510Ae27E2e1, 18),
+    ('lever-lever-finance', 'ethereum', 'LEVER', 0x4B5f49487ea7B3609b1aD05459BE420548789f1f, 18),
```

Comment:
it appears this id cannot be found on coinpaprika. please find the proper token on their site and locate the api id value in the info section. if it doesn't exist, we can't pull in here

Reply:
I checked before posting to see if they exist:
https://coinpaprika.com/coin/lever-leverfi/
https://coinpaprika.com/coin/hifi-hifi-finance/

Reply:
it should be `lever-leverfi` instead of `lever-lever-finance`

Reply:
Update lever-lever-finance to lever-leverfi